packages/ripple-nuxt-tide/modules/data-driven-component/components/TideMapPreventionFamilyViolence/TideMapPreventionFamilyViolence.vue

```diff
@@ -1,21 +1,44 @@
 <template>
   <div class="tide-map">
-    <h2 v-if="title"
-      class="tide-map__title">{{title}}</h2>
-    <RplMarkup
-      v-if="description"
-      :html="description" />
-    <pre v-if="err">
-      {{err}}
-    </pre>
-    <p class="rpl-visually-hidden">
-      The following is an interactive map component, showing the rough location of programs being run in support of the Prevention of Family Violence initiative.
-    </p>
-    <rpl-map
-      :baseMapUrl="baseMapUrl"
-      :customMethods="customMethods"
-      :minZoom="6"
-      />
+    <h2 v-if="title" class="tide-map__title">{{title}}</h2>
+    <RplMarkup v-if="description" :html="description"/>
+    <p
+      class="rpl-visually-hidden"
+    >The following is an interactive map component, showing the location of programs being run in support of the Prevention of Family Violence initiative.</p>
+    <div>
+      <div id="tide-map-container" class="tide-map-container">
```

Comment:
id won't be possible if multiple maps are used on the same page. Consider using refs and passing an element to toggleFullScreen.

Reply:
switched to refs